### DISCUSSION TITLE
Q-table column reorder

### DISCUSSION THREAD

**Discussion by simonmarien** [quasarframework/quasar#11535] [General - Components / Directives / etc] [answered]
When using a q-table filled with data rows, is it possible to change column order using javascript?

**Comment by MilosPaunovic** [comment, marked as answer]
This is the quickest example I could whip up, but it could set you in the right direction:
[https://codepen.io/milospaunovic/pen/NWaqRPm?editors=1011](https://codepen.io/milospaunovic/pen/NWaqRPm?editors=1011)
